python/ml4ir/base/model/scoring/interaction_model.py

```diff
@@ -35,9 +35,13 @@ def __call__(self, inputs: Dict[str, Input]):
             computing custom metrics and losses
         """
         train_features, metadata_features = self.feature_layer_op(inputs)
-        train_features, metadata_features = self.transform_features_op(
-            train_features, metadata_features
-        )
+        # separate the default learnable features from the fr_features
+        if self.feature_config.feature_exists('fr', trainable=True):
+            train_fr_features, _ = self.transform_features_op({'fr_feature':train_features['fr_feature']}, metadata_features)
```

Comment:
@mohazahran Can you explain why you are doing this?

Reply:
Don't want to merge the normal features with the positional bias features.